tests/improver-weighted-blending/07-lin_nonlin_args.bats

```diff
@@ -33,24 +33,23 @@
 
```

Comment:
Please could this test file be renamed to indicate that it is testing for an error.